apps/test/integration/levelSolutions/craft/codeconnection.js

```diff
@@ -0,0 +1,29 @@
+var TestResults = require('@cdo/apps/constants.js').TestResults;
+import blockUtils from '@cdo/apps/block_utils';
+
+let _ = require('lodash');
+
+let blankPlane = _.range(100).map(function () {
+  return "";
+});
+let actionPlane = _.clone(blankPlane);
+actionPlane[(4 * 10) + 6] = "sheep";
```

Comment:
I made this file to use it as an initial step for a testing but will bring it later with actual test codes